prereq_install.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+
+# functions
+function create_dir {
```

Comment:
wtf? why don't you just use `mkdir -p $target_dir`?..

Reply:
replaced